facultyai/mlflow-faculty
# Allow creating runs

This PR allows creating runs with `mlflow.start_run`.

To test this on a faculty server
- uninstall faculty to force a reinstallation
- install mlflow-faculty on this branch -- this should pull faculty master and mlflow master.

```py
import os
os.environ["MLFLOW_TRACKING_URI"] = "faculty:{}".format(os.environ["FACULTY_PROJECT_ID"])
import mlflow

mlflow.start_run()
```

Tested this against currently deployed atlas and it works.
